### PR TITLE
Martinsonis Macrobinsonski - Excessive amount of Fe, known commonly as 'iron', of considerable size, weight, and mass attached to the sternum of a human male specimen, by means of a leather aperture designed to extend the wearer's carrying capacity.

### PR DESCRIPTION
**Pull request by MalricB**
## About The Pull Request
MWS 'Big Iron' gun kit, alternate item for the HoS gun.
-Three different batteries, each with 7 shots:
*Ion
*Disabler
*Lethal
-Each battery goes into one of the three cylinder slots.
-Cylinder can be recharged using a recharger.
-Cylinder goes into gun. Blam blam.

https://imgur.com/JtdtmKS

## Why It's Good For The Game

i dunno it's 6:10 AM and I'm way too tired but funni revolver man go woosh

## Changelog
:cl:
add: Added MWS 'Big Iron' as a HoS gun
add: Added a weapon choice beacon (and a choice beacon in case someone else wants to make their own beacons)
del: my will to live
imageadd: added some icons to ammo.dmi and projectiles.dmi
/:cl:

### CODE REVIEW

**Comment by MalricB** [comment]
gotta add them in the sci tech as unlockables to be printed but my brain matter is at negative size rn

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+	var/obj/structure/closet/supplypod/bluespacepod/pod = new()
+	pod.explosionSize = list(0,0,0,0)
+	new_item.forceMove(pod)
+	var/msg = "<span class='danger'>After making your selection, you notice a strange target on the ground. It might be best to step back!</span>"
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(istype(H.ears, /obj/item/radio/headset))
+			msg = "You hear something crackle in your ears for a moment before a voice speaks.  \"Please stand by for a message from Central Command.  Message as follows: <span class='bold'>Item request received. Your package is inbound, please stand back from the landing site.</span> Message ends.\""
+	to_chat(M, msg)
+	new /obj/effect/DPtarget(get_turf(src), pod)
```
Comment (ArchieBeepBoop) [inline code/game/objects/items/miscellaneous.dm:50]: This is not defined and causes build errors

Comment (MalricB) [reply code/game/objects/items/miscellaneous.dm:50]: nani
do you mena DPtarget

Comment (MalricB) [reply code/game/objects/items/miscellaneous.dm:50]: because it is defined im not sure what you mean ;w;
https://i.imgur.com/CMCB2bD.png

Comment (ArchieBeepBoop) [reply code/game/objects/items/miscellaneous.dm:50]: effect/DPtarget does not exist in hyper
image

You're looking for

new /obj/effect/abstract/DPtarget instead

Comment (ArchieBeepBoop) [reply code/game/objects/items/miscellaneous.dm:50]: DM compiler will likely not complain about undefined typepaths like that and simply not render them, but VScode will.

Comment (MalricB) [reply code/game/objects/items/miscellaneous.dm:50]: kk it's here https://github.com/quotefox/Hyper-Station-13/pull/903

**Pull request merged**